deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml

```diff
@@ -34,6 +34,8 @@ spec:
           value: "3"
         - name: skaha.sessionexpiry
           value: "345600"
+        - name: skaha.defaultquotagb
+          value: "10"
```

Comment:
I think the actual default we use in prod is bigger.  @kcasteels ?

Reply:
This was the number that was hardcoded in the Java class, though.

Reply:
(in base only)